week0004/truongngodang/java/Week0004.java

```diff
@@ -0,0 +1,152 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Week0004 {
+
+    public static void main(String[] args) {
+
+        if (ReaderCVS.checkInputs(args)) {
+            ArrayList<Guest> guestsEvent1 = ReaderCVS.read(args[0]);
+            ArrayList<Guest> guestsEvent2 = ReaderCVS.read(args[1]);
+            Response response = new Response();
+            Checker checker = new Checker();
+            checker.check(guestsEvent1, guestsEvent2);
+            System.out.println(response.standardPrint(checker.getGuestErrors()));
+        } else {
+            System.out.println("File Not Allow!");
+        }
+    }
+}
+
+class Guest {
+    private String fname;
+    private String lname;
+    private String email;
+
+    public Guest(String fname, String lname, String email) {
+        this.fname = fname;
+        this.lname = lname;
+        this.email = email;
+    }
+
+    public Guest() {
+    }
+
+    public String getFname() {
+        return fname;
+    }
+
+    public void setFname(String fname) {
+        this.fname = fname;
+    }
+
+    public String getLname() {
+        return lname;
+    }
+
+    public void setLname(String lname) {
+        this.lname = lname;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+}
+
+class ReaderCVS {
+
+    public static ArrayList<Guest> read(String csvFilename) {
+        String line = "";
+        String cvsSplitBy = ",";
+        ArrayList<Guest> guests = new ArrayList<Guest>();
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(new File(csvFilename)));
+            for (int i = 0; (line = br.readLine()) != null; i++) {
+                if (i > 0) {
+                    String[] infos = line.split(cvsSplitBy);
+                    guests.add(new Guest(infos[0], infos[1], infos[2]));
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return guests;
+    }
+
+    public static boolean check(String fileName) {
```

Comment:
I don't think we need to do this. I asked to check that the size of argv is 2 and the filenames are of valid files.